src/d/lyt/d_lyt_map_markers.cpp

```diff
@@ -0,0 +1,1926 @@
+#include "d/lyt/d_lyt_map_markers.h"
+
+#include "common.h"
+#include "d/a/d_a_base.h"
+#include "d/a/d_a_item.h"
+#include "d/a/d_a_player.h"
+#include "d/a/obj/d_a_obj_base.h"
+#include "d/a/obj/d_a_obj_save.h"
+#include "d/a/obj/d_a_obj_tbox.h"
+#include "d/d_cs_base.h"
+#include "d/d_cursor_hit_check.h"
+#include "d/d_gfx.h"
+#include "d/d_message.h"
+#include "d/d_player.h"
+#include "d/d_player_act.h"
+#include "d/d_sc_game.h"
+#include "d/d_stage.h"
+#include "d/d_stage_mgr.h"
+#include "d/flag/dungeonflag_manager.h"
+#include "d/flag/storyflag_manager.h"
+#include "d/lyt/d2d.h"
+#include "d/lyt/d_lyt_map.h"
+#include "d/lyt/d_lyt_map_global.h"
+#include "d/lyt/d_window.h"
+#include "d/snd/d_snd_small_effect_mgr.h"
+#include "f/f_base.h"
+#include "f/f_manager.h"
+#include "m/m_vec.h"
+#include "nw4r/lyt/lyt_bounding.h"
+#include "nw4r/lyt/lyt_pane.h"
+#include "nw4r/lyt/lyt_types.h"
+#include "sized_string.h"
+#include "toBeSorted/arc_managers/layout_arc_manager.h"
+
+inline mVec3_c vec2ToVec3XY(const mVec2_c &v) {
+    return mVec3_c(v.x, v.y, 0.0f);
+}
```

Comment:
Should this be part of the mVec heaader?

Reply:
Let's address it with #259.